client/src/pages/SignIn.js

```diff
@@ -0,0 +1,150 @@
+      backgroundColor: "#FF510C",
+    },
+  },
+  leftSideContainer: {
+    width: "50%",
+    height: "100vh",
+    float: "left",
+  },
+  titleLogo: {
+    width: "50%",
+    marginLeft: theme.spacing(20),
+    paddingBottom: theme.spacing(2),
+    textAlign: "center",
+  },
+  rightSideContainer: {
+    width: "50%",
+    height: "100vh",
+    [theme.breakpoints.up("md")]: {
+      backgroundImage: `url(${BackgroundImg})`,
+      backgroundSize: "cover",
+      backgroundPosition: "left",
+    },
+    float: "left",
+  },
+  textField: {
+    margin: theme.spacing(2),
+    width: "30ch",
+  },
+  whiteText: {
+    color: "white",
+  },
+  marginLeft_2: {
+    margin: "2em",
+  },
+});
+
+class SignInPage extends Component {
+  state = {
+    email: "",
+    password: "",
+  };
+
+  loadSignUp = () =>
+    ReactDOM.render(<SignUp />, document.getElementById("root"));
+
+  render() {
+    const { classes } = this.props;
+    return (
+      <div className={classes.signInContainer}>
+        <div className={classes.leftSideContainer}>
+          <div className={classes.titleLogo}>
+            <Typography variant="h4">CHEF'S MENU</Typography>
+          </div>
+          <Card className={classes.card}>
+            <CardContent>
+              <Typography className={classes.title} gutterBottom>
+                Login
+              </Typography>
+              <form autoComplete="off">
+                <Box display="flex">
+                  <TextField
+                    className={classes.textField}
+                    required
+                    name="signupEmail"
+                    label="Email"
+                    placeholder="Enter your e-mail address"
+                    variant="outlined"
+                  />
+                </Box>
+                <Box display="flex">
+                  <TextField
+                    className={classes.textField}
+                    required
+                    type="password"
+                    name="signupPwd"
+                    label="Password"
+                    placeholder="Enter password"
+                    variant="outlined"
+                  />
+                </Box>
+              </form>
+            </CardContent>
+            <CardActions style={{ justifyContent: "center" }}>
+              <Button type="submit" className={classes.button}>
+                Sign In
+              </Button>
+            </CardActions>
+          </Card>
+        </div>
+        <div className={classes.rightSideContainer}>
+          <div style={{ padding: "25px" }}>
+            <Typography
+              className={classes.whiteText}
+              variant="h6"
+              align="right"
+            >
+              Don't have an account ?
+              <Button
+                className={`${classes.button} ${classes.marginLeft_2}`}
+                variant="contained"
+                onClick={this.loadSignUp}
+              >
+                Sign Up
+              </Button>
```

Comment:
Wrap this in a `<Link to="/signup"></Link>` with the `Link` component from `react-router-dom`

Reply:
Done.